### DISCUSSION TITLE
bulk elasticsearch post

### DISCUSSION THREAD

**Discussion by AlexDF2017** [dmachard/DNS-collector#371] [Q&A] [answered]
Hi all, we have a huge amount of data coming from Unbound and targeted to Elasticsearch, the problem is that Dnscollector is pushing data using simple POST (one POST for every record), and this is very inefficient, so the Elasticsearch target is receiving datas related to hours in the past.
Is it possible for Dnscollector to inject using a bulk POST, for example sending 100 records at once? AlexDF.

**Comment by dmachard** [comment]
Hi, It should be easy to implement because this mechanism is already implemented in others loggers.
Can you open an issue to track this new feature ?

**Reply by AlexDF2017** [reply, marked as answer]
Opened issue: Elasticsearch bulk insert #374.
Thanks
